__init__.py

```diff
@@ -31,8 +31,10 @@ def __init__(self):
         super(PairingSkill, self).__init__("PairingSkill")
         self.api = DeviceApi()
         self.data = None
+        self.last_request = None
         self.state = str(uuid4())
         self.delay = 10
+        self.expiration = 72000
```

Comment:
Add a comment telling what this value means (looks like a magic number).  Something like:

self.expiration = 72000  # 20 hours